ee/desktop/runtime/runner_posix.go

```diff
@@ -0,0 +1,17 @@
+//go:build darwin || linux
```

Comment:
is the name of this file `runner_posix.go` appropriate? Was trying to find something other than `runner_notwindows.go`

Reply:
Um, probably. Sure!

But I'd name it `dialer_xxxxx.go` (or `dial_context.go`), since it's just the dial function

consider maybe instead of calling `desktop.DesktopSocketPath(pid)` to pass the pid as an argument. (Or likely, as suggested above, launcher will set the path, and handoff to desktop anyhow)